### ISSUE TITLE
New Task: For raw ATN profiling datasets, finalize and implement automated NCEI data archive pathway and archive administrative tools.

### ISSUE THREAD

**Issue by MathewBiddle** [ioos/marine_life_data_network#42]
### Who is requesting this?

@ioos/marine-life, and the ATN community

### What is being requested?

- Establish netCDF template in coordination with ATN DC, IOOS National Glider Data Assembly Center (POC: @kbailey-noaa) and IOOS DMAC (POC: @mwengren ).
- final agreement of nc metadata attributes with NCEI
- create metadata and format file template for NCEI review
- test load template to NCEI ftp site/ WAF
- begin operationally submitting non-embargoed deployments to NCEI
- create automated service within ATN DAC Reg App for grouping deployments into synthetic 'projects' and connecting metadata returned from NCEI for access by PIs
- Ensure archived data, dataset citations and DOIs are integrated into ATN DAC portal catalog 
- Update documentation at https://ioos.github.io/ioos-atn-data/atn-sat-telem-specification-v1-0.html to reflect current netCDF implementation. 
- Document issues and proposals for future versions at https://github.com/ioos/ioos-atn-data/issues

### What is the current status quo (i.e., what happens if this does not get done)?

ATN profiling data are not archived at NCEI and PARR requirements are not met.

### What indicates this is done (i.e., how do we know this is complete)?

- [ ] NCEI automatically pulls data from ATN on identified frequency
- [ ] IOOS publishes a news story about it.

### Provide a description or any other important information.

xref:
* https://github.com/ioos/marine_life_data_network/issues/18
* https://github.com/ioos/marine_life_data_network/issues/41

**Comment by MathewBiddle** [comment]
What does "and archive administrative tools" mean in the title?

**Comment by mmckinzie** [comment]
I am also not sure what 'archive administrative tools' means...maybe remove from title?

**Comment by MathewBiddle** [comment]
😆 okay if we both don't know what it means, let's remove it.

**Comment by conniekot** [comment]
@iwensu0313 Please update this issue with progress, if it has been incorporated into https://github.com/ioos/ioos_qc, and any links to the code - thanks!

**Comment by iwensu0313** [comment]
Creating a NCEI archival template for profile data will start in March 2025 in the new contract. Active NCEI archival pipeline is  captured here for ATN trajectory data https://github.com/ioos/marine_life_data_network/issues/41 - we will be able to continue archiving trajectory data to NCEI as they come in.

**Comment by iwensu0313** [comment]
Is this ticket blocked by https://github.com/ioos/marine_life_data_network/issues/57?

**Comment by MathewBiddle** [comment]
@mmckinzie just needs to review. Should be the same as the trajectory template.

**Comment by mmckinzie** [comment]
I've begun discussing the need to generate test files based on the profile template with Tetra Tech during our biweekly coordinator call. Generating and reviewing these files to ensure the .nc files are consistently generated as expected and without error will need to happen before we can fully finalize and implement the NCEI pipeline for these profile data. A process that will likely take several weeks. I recommend we keep this task blocked until we're closer to full implementation or at least until the first test files are generated.

**Comment by MathewBiddle** [comment]
Why several weeks? We need to prioritize this.

**Comment by mmckinzie** [comment]
I agree it needs to be prioritized, but it took many months to finalize the process for generating the trajectory files once Tetra Tech started supplying example files for review. I anticipate that finalizing the profile data files will be much faster, but will still likely take weeks to complete before we're ready to finalize and implement the NCEI pipeline to serve these data for integration into the ATN collection. It's that review process I suspect will take some time, not the implementation of the NCEI pipeline, which will happen very quickly.

**Comment by iwensu0313** [comment]
Just to confirm. These are the high level steps right?
1. Megan to work w/ relevant parties to finalize profile template (IN PROGRESS) https://github.com/ioos/marine_life_data_network/issues/57
2. Tetra Tech to generate profile .nc files in accordance w/ the profile template finalized above; likely prepped in a clean staging environment for easy review
3. Review process: Megan, NCEI, IOOS, others review of profile files; Tetra Tech to iterate on files / workflow as needed
4. Once review is done, Tetra Tech to finalize and implement workflow to push profile data to NCEI WAF

I see step 2 onwards as part of this ticket. so once step 1 is done, let's plan to unblock this ticket so I know that it's something we need to move on. Or if step 2 is part of this ticket https://github.com/ioos/marine_life_data_network/issues/57 it will help if I can add myself as an assignee so I can keep track of what Tetra tech needs to do and when, and if someone is waiting on us.

**Comment by mmckinzie** [comment]
@iwensu0313 I also see step 2 onwards as being associated with this ticket. I hope to have issue #57 (step 1) finalized by early next week. Once that is done, we should unblock this ticket.
